### PR TITLE
Chore/adjust btn placement and username overflow/noticket

### PR DESCRIPTION
**Pull request by alexisBimatovs**
## store pr with details & screenshots
https://github.com/forever-inc/a-new-hope/pull/1155

### CODE REVIEW

**Comment by alexisBimatovs** [comment]
history looks a little funky here because I branched off of master, will use dev moving forward, assumed they were in sync

File: app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss [hunk 1]
```diff
@@ -104,6 +104,12 @@ $padding-small-horizontal: 3px;
   text-align: left;
 }
 
+.user_info-logged_in-account_dropdown-storage-name {
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss]: why does this class name forego the .navigation- module convention?

File: app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss [hunk 2]
```diff
@@ -104,6 +104,12 @@ $padding-small-horizontal: 3px;
...
+  display: inline-block;
+  max-width: 200px; // Req for Safari
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss]: prefer variable here

File: app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss (hunk 1 above)
Comment (alexisBimatovs) [reply app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss]: I was following the class name convention of the parent element

screen shot 2018-02-05 at 10 20 25 am

File: app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss (hunk 2 above)
Comment (alexisBimatovs) [reply app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss]: I replaced with variable but prefer to keep comment so it is clear it is required for Safari, I feel like if testing in Chrome someone could think of this is not necessary without getting into cross-browser testing and I don't think adding the context of it being used for Safari in the variable name is appropriate either

**Comment by schweitzdesign** [comment]
LGTM

**Pull request merged**